chain/ethereum/src/ethereum_adapter.rs

```diff
@@ -1895,3 +1892,125 @@ fn resolve_transaction_receipt(
         }
     }
 }
+
+/// Retrieves logs and the associated transact receipts, if required by the [`EthereumLogFilter`].
+async fn get_logs_and_transactions(
```

Comment:
Minor thing, the comments in this function should start with upper case instead of lower case.

Reply:
Thanks! Fixed in 290529777